packages/contracts/docs/core/01-how-it-works/01-core/02-permissions/index.md

```diff
@@ -0,0 +1,153 @@
+  bytes32 _permissionId
+) internal pure returns (bytes32) {
+  return keccak256(abi.encodePacked('PERMISSION', _who, _where, _permissionId));
+}
+```
+
+This concatenated information is then stored as `keccak256` hashes inside a mapping like this one:
+
+```solidity title="contracts/core/permission/PermissionManager.sol"
+mapping(bytes32 => address) internal permissionsHashed;
+```
+
+Here, the `bytes32` keys are the permission hashes and the `address` values are either zero-address flags, such as `ALLOW_FLAG = address(0)` and `UNSET_FLAG = address(2)` indicating if the permission is set, or an actual address pointing to a `PermissionCondition` contract, which is discussed in the next section of this guide.
+
+### Authorization Modifiers
+
+Using **authorization modifiers** is how we make functions permissioned. Permissions are associated with functions by adding the `auth` modifier, which includes the permission identifier in the function’s definition header.
+
+For example, one can call the `execute` function in the DAO when the address making the call has been granted the `EXECUTE_PERMISSION_ID` permission.
+
+```solidity title="contracts/core/DAO.sol"
+function execute(
+  bytes32 callId,
+  Action[] calldata _actions,
+  uint256 allowFailureMap
+)
+  external
+  override
+  auth(address(this), EXECUTE_PERMISSION_ID)
+  returns (bytes[] memory execResults, uint256 failureMap);
+```
+
+### Managing Permissions
+
+To manage permissions, the DAO contract has the `grant`, `revoke` and `grantWithCondition` functions in its public interface.
+
+#### Granting and Revoking Permissions
+
+The `grant` and `revoke` functions are the main functions we use to manage permissions.
+Both receive the `_permissionId` identifier of the permission and the `_where` and `_who` addresses as arguments.
+
+```solidity title="contracts/core/permission/PermissionManager.sol"
+function grant(
+  address _where,
+  address _who,
+  bytes32 _permissionId
+) external auth(_where, ROOT_PERMISSION_ID);
+```
+
+To prevent these functions from being called by any address, they are themselves permissioned via the `auth` modifier and require the caller to have the `ROOT_PERMISSION_ID` permission in order to call them.
+
+:::note
+By default, the `ROOT_PERMISSION_ID` permission is granted only to the `DAO` contract itself. Contracts related to the Aragon infrastructure temporarily require it during the [DAO creation](../../02-framework/01-dao-creation-process.md) and [plugin setup ](../../02-framework/02-plugin-repository/04-plugin-setup.md) processes.
+:::note
+
+This means, that these functions can only be called through the DAO’s `execute` function that, in turn, requires the calling address to have the `EXECUTE_PERMISSION_ID` permission.
+
+:::note
+By default, the `EXECUTE_PERMISSION_ID` permission is granted to governance contracts (such as a majority voting plugin owned by the DAO or a multi-sig). Accordingly, a proposal is often required to change permissions.
```

Comment:
I wouldn't say "by default". What the app does by default is not relevant here.

I would say something like "typically, such a permission is granted to a governance plugin"   or similar

Reply:
Changed in 7bb7f54e5244cde48b71b1d699056b189e9e4349